frontend/src/assets/_bazarr.scss

```diff
@@ -52,6 +52,17 @@ $header-height: 64px;
     overflow-wrap: anywhere;
   }
 
+  .table-progress-label {
+    position: absolute;
+    top: 0;
+    left: 50%;
+    transform: translateX(-50%);
+    color: rgb(232, 230, 227);
+    line-height: 1.55;
+    font-size: calc(1rem * 0.65);
+    font-weight: 700;
+  }
```

Comment:
This should no be in `_bazarr.scss` as it is not global.

You should instead use styles API of mantine.

https://mantine.dev/core/progress/?t=styles-api

<Progress.label styles={{ .mantine-Progress-label: ...}} /`.

Reply:
Im not sure if i understand, something like this?
```js
              {episodeMissingCount === episodeFileCount && <Progress.Label style={{
                position: 'absolute',
                top: '3px',
                left: '50%',
                transform: 'translateX(-50%)'
              }}>{label}</Progress.Label>}
```
This looks the same as above, should i use a scss module for those inline styles?

Reply:
No this is wrong. You should use `styles` and not style.

Reply:
Let me pick an example to you.

Reply:
Check `pages/views/ItemOverview.tsx`

Reply:
Somethink like this is ok? Or should i move it to a module.scss like in that example?
```js
{episodeMissingCount === episodeFileCount && <Progress.Label styles={{
  label: {
    position: 'absolute',
    top: '3px',
    left: '50%',
    transform: 'translateX(-50%)'
  }
}}>{label}</Progress.Label>}
```
Sorry, I never used Mantine before

Reply:
You can leave inline for now, I will refactor the modules properly later, now the series filename is called index.tsx, that will cause issues creating a module css for that.

Reply:
> Somethink like this is ok? Or should i move it to a module.scss like in that example?

Your example looks good for now! Moving to a module.css might have more complications, i will work on refactor this on the future.

Reply:
pushed, thanks